release_util/management/commands/run_migrations.py

```diff
@@ -0,0 +1,179 @@
+import sys
+import traceback
+import yaml
+from cStringIO import StringIO
+from timeit import default_timer
+from collections import defaultdict
+from copy import deepcopy
+
+from django.core.management import call_command, CommandError
+from django.core.management.base import BaseCommand
+from django.db.utils import DatabaseError
+
+
+class MigrationSession(object):
+    """
+    Class which is initialized with Django app/model migrations to perform.
+    Performs migrations while keeping track of the state of each migration.
+    Provides the state of all migrations on demand.
+    """
+    def __init__(self, input_yaml, stderr):
+        self.to_apply = []
+        self.migration_state = {
+            'success': defaultdict(dict),
+            'failure': defaultdict(dict),
+            'unapplied': [],
+            'rollback_commands': [],
+        }
+        self.timer = default_timer
+        self.stderr = stderr
+
+        # Load the passed-in YAML into a dictionary.
+        self.input_migrations = yaml.load(input_yaml)
```

Comment:
If you get rid of the tuples in the yaml output, you should switch any load statements to safe_load as pyyaml will execute any python code that is found in the input à-la pickle.